style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
- [ OPTIONAL ] It's great working on the styles but I recomend to change your file name to be index.css as required in the requirements to refactor your code 🙏🏻🔥♻✅